app/src/main/java/nodomain/freeyourgadget/gadgetbridge/service/DeviceCommunicationService.java

```diff
@@ -174,7 +174,7 @@ public void onReceive(Context context, Intent intent) {
                         try (DBHandler dbHandler = GBApplication.acquireDB()) {
                             DaoSession session = dbHandler.getDaoSession();
                             boolean askForDBMigration = false;
-                            if (DBHelper.findDevice(device, session) == null && device.getType() != DeviceType.VIBRATISSIMO) {
+                            if (DBHelper.findDevice(device, session) == null && device.getType() != DeviceType.VIBRATISSIMO && (device.getType() != DeviceType.LIVEVIEW)) {
```

Comment:
Also I am to blame.
I didn't even think about the fact that support for new devices does not magically appear in old versions :laughing: 

I rather thought that about vibratissimo not having any activity data... except .... a temperature sensor hmmm...